django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -216,8 +225,7 @@ def save_message(
     @database_sync_to_async
     def get_ai_settings(chat: Chat) -> AISettings:
         ai_settings = model_to_dict(chat.user.ai_settings, exclude=["label", "chat_backend"])
-        chat_backend = model_to_dict(chat.user.ai_settings.chat_backend)
-        ai_settings["chat_backend"] = chat_backend
+        ai_settings["chat_backend"] = model_to_dict(chat.chat_backend)
```

Comment:
we want to send the `chat_backend` belonging to the current chat not to the `ai_settings`